core/src/main/java/org/aerogear/mobile/core/metrics/publisher/LoggerMetricsPublisher.java

```diff
@@ -0,0 +1,26 @@
+package org.aerogear.mobile.core.metrics.publisher;
+
+import org.aerogear.mobile.core.logging.Logger;
+import org.aerogear.mobile.core.metrics.Metrics;
+import org.aerogear.mobile.core.metrics.MetricsPublisher;
+
+/**
+ * All metrics data will be logged only
+ */
+public final class LoggerMetricsPublisher implements MetricsPublisher {
+
+    private final Logger logger;
+
+    public LoggerMetricsPublisher(Logger logger) {
+        this.logger = logger;
+    }
+
+    @Override
+    public void publish(Metrics... metrics) {
+        logger.info("Metrics collected");
```

Comment:
not sure if we should use `info` level here. The message is not very clear as it could be the metrics is being collected on the server side.

Also given that the real metrics data are only logged at `debug` level, which means they will not be printed for a release app,  not sure if there is any value to print the message here?